boskos/README.md

```diff
@@ -0,0 +1,51 @@
+# WIP boskos
+
+
+## Background
+[βοσκός](https://en.wiktionary.org/wiki/%CE%B2%CE%BF%CF%83%CE%BA%CF%8C%CF%82) - cattle-master in greek!
+
+boskos is a http service that allow k8s test jobs to run independently from GCP projects.
```

Comment:
That sounds like a slower approach given the potential amount of request at the same time.

I would ultimately like to change the PD to use CloudSQL, so that we can have multiple replicas of the http service. Rely on a GCS object seems not a good synchronization pattern.